zrml/rikiddo/fuzz/rikiddo_pallet.rs

```diff
@@ -56,8 +57,11 @@ fuzz_target!(|data: Data| {
             if idx % 2 == 1 {
                 current_block += 1;
                 run_to_block(current_block);
-                let _ = <Runtime as Config>::Timestamp::set(RawOrigin::None.into(), current_block)
-                    .unwrap();
+                let _ = <Runtime as Config>::Timestamp::set(
+                    RawOrigin::None.into(),
+                    MinimumPeriod::get() * current_block,
+                )
+                .unwrap();
```

Comment:
@sea212 I don't think this modification changes any testing logic but can you confirm?

Reply:
It does change the testing logic. Since in the first case where `Timestamp::set` is called the number of seconds is greater than 1, both ema instance (long and short) skip both in the last state and afterwards represent the same value. This leads to the situation that the fee parameter will always be 1.
Why did you add this in the first place? The fuzz test does not have to be logically consistent with our parameters, it must be able to test as many code execution paths with test data sets. Can you please revert the changes for this file and remove zeitgeist-primitives again?

Reply:
> Why did you add this in the first place?

`MinimumPeriod` is not 0 on production environments, CI fails without this change (https://github.com/zeitgeistpm/zeitgeist/runs/3635598436?check_suite_focus=true) and parameters are being de-duplicated  as previously stated.

> Can you please revert the changes for this file and remove zeitgeist-primitives again?

Yes, I will revert